contents/docs/user-guides/groups.md

```diff
@@ -0,0 +1,296 @@
+---
```

Comment:
QQ: We call this feature internally group analytics which also translates better to SEO - I think we should call this doc the same.